scripts/image_tools/equirectangular_to_cubemap.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 1996-2019 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Convert an image having an equirectanglar projection to 6 cubemap images."""
+# Support HDR, PNG and JPG formats.
+
+# Reference about the projection:
+# - https://stackoverflow.com/a/36976448
+
+import optparse
```

Comment:
Just a side note for the future script, the `optparse` module is deprecated we should use `argparser` instead (work the sam way).